deploy/helm/sumologic/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: sumologic
-version: 2.0.5
-appVersion: 2.0.5
+version: 2.0.6
```

Comment:
Don't we need a `rc` version?

Reply:
There are mostly doc changes.
The only real fix is for setup job - and we run this with `v2.1` already so I don't feel like we do need `rc.1` for that.
I'm OFC open for discussion :)

Reply:
Ok for taking into account the diff/log